NEWS.md

```diff
@@ -1,3 +1,9 @@
+# version 3.0
+
+## Breaking changes
+
+- RegisterQD is being updated to incorporate GPU processing. Use RegisterMismatch before RegisterQD for CPU methods, and RegisterMismatchCuda for GPU methods. Note that loading both will cause method conflicts. 
```

Comment:
```suggestion
- RegisterQD now requires an extra step because of enhancements that support either CPU and GPU processing. For CPU processing (formerly the only option), now you must manually load the [RegisterMismatch package](https://github.com/HolyLab/RegisterMismatch.jl): `using RegisterMismatch, RegisterQD`. For GPU processing, you should instead load the [RegisterMismatchCuda package](https://github.com/HolyLab/RegisterMismatchCuda.jl): `using RegisterMismatchCuda, RegisterQD`. *Note that loading both mismatch packages in the same session will cause method conflicts.* Both mismatch packages are registered in the publicly-available [HolyLabRegistry](https://github.com/HolyLab/HolyLabRegistry), and users are advised to add that registry. 
```

Reply:
Do you mind if I shuffle these suggested changes around? I think that it would make more sense if we add the more explicit instructions directly to the README instead of to NEWS, since new users are more likely to need this information and also more likely to just look at the readme.

Reply:
That makes sense. I added them in the order in which I read, and your proposal is better.